src/main/resources/org/jenkinsci/plugins/github_branch_source/app_credentials/MigrationAdminMonitor/message.jelly

```diff
@@ -0,0 +1,17 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form">
+    <div class="alert alert-warning">
+        <form id="${it.id}" method="post" action="${rootURL}/${it.url}/act" name="${it.id}">
+            <f:submit name="clear" value="${%Dismiss}"/>
+        </form>
+        ${%blurb}
+        <p><a href="TODO">See here for more information.</a></p>
+        <p>${%affectedCredentials}
+            <ul>
+                <j:forEach var="c" items="${it.migratedCredentialIds}">
+                    <li>${c}</li>
+                </j:forEach>
+            </ul>
```

Comment:
Perhaps this needs to be in a `<details>` tag or have fixed height so that it doesn't take up the whole page if there are a lot of credentials.